website/docs/api/docusaurus.config.js.md

```diff
@@ -429,6 +429,30 @@ module.exports = {
 };
 ```
 
+### `headTags` {#headTags}
+
+An array of tags that will be inserted in the HTML `<head>`. The values must be objects that contain two properties; `tagName` and `attributes`. `tagName` must be a string that determines the tag being created; eg `"link"`. `attributes` must be an attribute-value map.
+
+- Type: `{ tagName: string; attributes: Object; }[]`
+
+Example:
+
+```js title="docusaurus.config.js"
+module.exports = {
+  headTags: [
+    {
+      tagName: 'link',
+      attributes: {
+        rel: 'monetization',
+        href: 'https://ilp.uphold.com/LwQQhXdpwxeJ',
```

Comment:
same, we can't have this url in our doc 😅

Also I'd rather have another example that is an accepted proposal and a commonly used case (ie not WIP monetization API)

For example:

```js
{
  tagName: "link",
  attributes: { rel: "icon", href: "img/docusaurus.png" }
};
```

Reply:
I'll amend to this!